frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -1,198 +1,213 @@
-          <Typography
-            style={{ lineHeight: "1.3", margin: "-5px 0px 10px 10px" }}
-            component="span"
-            color="primary"
-            data-qa="title"
-          >
-            <NavigationLink to="/" onClick={onNavigate}>
-              <GroupHeading>clutch</GroupHeading>
-            </NavigationLink>
-            <Typography variant="caption">Welcome {userId()}!</Typography>
-          </Typography>
-        </Grid>
-      </Grid>
-    </DrawerHeader>
-  );
-};
-
-const Divider = styled(MuiDivider)`
-  ${({ theme }) => `
-  background-color: ${theme.palette.accent.main};
-  `}
-`;
+// sidebar
+const drawerWidth = "100px";
+
+const DrawerPanel = styled(MuiDrawer)({
+  flexShrink: 0,
+  width: `${drawerWidth}`,
+  "div[class*='MuiDrawer-paper']": {
+    width: `${drawerWidth}`,
+    top: "64px",
+    backgroundColor: "linear-gradient(0deg, #FFFFFF, #FFFFFF), #FFFFFF",
+    boxShadow: "0px 5px 15px rgba(53, 72, 212, 0.2)",
+  },
+});
+
+// sidebar groupings
+const GroupList = styled(List)({
+  padding: "0px",
+});
+
+const GroupListItem = styled(ListItem)({
+  flexDirection: "column",
+  height: "82px",
+  padding: "16px 8px 0px 8px",
+  "&:hover": {
+    backgroundColor: "#E7E7EA",
+  },
+  "&:active": {
+    backgroundColor:
+      "linear-gradient(0deg, rgba(255, 255, 255, 0.85), rgba(255, 255, 255, 0.85)), #0D1030",
+  },
+  "&.Mui-selected": {
+    backgroundColor: "#EBEDFB",
+    ".MuiAvatar-root": {
+      color: "#FFFFFF",
+      background: "#3548D4",
+    },
+  },
+});
+
+const Avatar = styled(MuiAvatar)({
+  color: "rgba(13, 16, 48, 0.38)",
+  background: "rgba(13, 16, 48, 0.12)",
+  height: "36px",
+  width: "36px",
+});
+
+const GroupHeading = styled(Typography)({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontWeight: 500,
+  fontSize: "14px",
+  lineHeight: "18px",
+  flexGrow: 1,
+  paddingTop: "8px",
+  paddingBottom: "8px",
+});
+
+// sidebar submenu
+const Popper = styled(MuiPopper)({
+  zIndex: 1200,
+  paddingTop: "16px",
+});
+
+const Paper = styled(MuiPaper)({
+  minWidth: "230px",
+  border: "1px solid #E7E7EA",
+  boxShadow: "0px 10px 24px rgba(35, 48, 143, 0.3)",
+});
+
+// sudebar submenu groupings
+const LinkListItem = styled(ListItem)({
+  backgroundColor: "#FFFFFF",
+  "&.Mui-selected": {
+    backgroundColor: "#EBEDFB",
+    ".MuiTypography-root": {
+      color: "#3548D4",
+    },
+  },
+  "&:hover": {
+    backgroundColor: "#E7E7EA",
+  },
+  height: "48px",
+});
+
+const LinkHeading = styled(Typography)({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontWeight: 500,
+  fontSize: "14px",
+  lineHeight: "18px",
+});
 
 interface GroupProps {
   heading: string;
   open: boolean;
   updateOpenGroup: (heading: string) => void;
-  onNavigate: () => void;
 }
 
-const Group: React.FC<GroupProps> = ({
-  heading,
-  open = false,
-  updateOpenGroup,
-  onNavigate,
-  children,
-}) => {
+const Group: React.FC<GroupProps> = ({ heading, open = false, updateOpenGroup, children }) => {
```

Comment:
there's a design bug - if you navigate for a workflow back to the clutch landing (by clicking the logo), then the sidebar still highlights the last active workflow. I've tried setting the selected state through React.useState instead but haven't gotten it to work yet. still working on fixing it

<img width="358" alt="Screen Shot 2020-12-10 at 9 15 22 AM" src="https://user-images.githubusercontent.com/39421794/101783535-63f80180-3ac8-11eb-828d-17060aa26c2b.png">